### DISCUSSION TITLE
ServiceLookupEditor usage issues!

### DISCUSSION THREAD

**Discussion by isadam** [serenity-is/Serenity#5300] [Help]
Dear volkanceylan  

I am trying to Use ServiceLookupEditor Filter  in Grid.ts  but I am getting Error 
Please give me Any Idea 

 filters.push({
                type: Serenity.ServiceLookupEditor,
                options: {
                    service: "Movie/Product/List",
                    cascadeFrom: "AccountId",
                    cascadeField: "AccountId"
                },
                field: fld.Id,
                title: 'Account', 

I didn't got any Example for your side how to user ServiceLookupEditor  in Quick Filter using Type Script  (Grid.ts)

Please update me if you have any idea 

Thanks

_Originally posted by @Weblife123 in https://github.com/serenity-is/Serenity/issues/4570#issuecomment-707718737_

**Comment by VictorTomaili** [comment]
I transfer this question to discussions.
